pavelib/paver_tests/test_stylelint.py

```diff
@@ -13,11 +13,6 @@ class TestPaverStylelint(PaverTestCase):
     """
     Tests for Paver's Stylelint tasks.
     """
-
-    def setUp(self):
-        super(TestPaverStylelint, self).setUp()
-        pass
```

Comment:
Sigh...this is the kind of thing that makes you want to look in the history and see just how it ended up there in the first place.